pkg/config/config.go

```diff
@@ -99,17 +99,20 @@ func Empty() *Config {
 // Keys returns the possible config keys. TODO: make all the keys constants.
```

Comment:
How about solving this TODO?

Reply:
I've left a comment for the keys below who don't have constants, they shouldn't belong to this package in the first place so we didn't create constants for them.